inc/Mailer.class.php

```diff
@@ -754,6 +755,13 @@ public function send() {
                 $headers .= MAILHEADER_EOL.trim($data['headers']);
             }
 
+            if(!function_exists('mail')){
+                $emsg = $lang['email_fail'] . $subject;
+                error_log($emsg);
+                msg($emsg,-1,__LINE__, __FILE__, MSG_MANAGERS_ONLY);
```

Comment:
Since you have dynamic `$subject` here this might needs HTML encoding:

```suggestion
                msg(hsc($emsg), -1, __LINE__, __FILE__, MSG_MANAGERS_ONLY);
```